ddtrace/compat.py

```diff
@@ -79,6 +79,25 @@ def to_unicode(s):
     msgpack_type = bytes
     numeric_types = (int, float)
 
+if PY2:
+    # avoids Python 3 `SyntaxError`
+    # this block will be replaced with the `six` library
+    from .utils.reraise import _reraise as reraise
+else:
+    def reraise(tp, value, tb=None):
```

Comment:
The function used here is from the `six` library: [six.py#L687-L696](https://github.com/benjaminp/six/blob/edb7d0051202280d44939fd48863d244fe7b2c2b/six.py#L687-L696)